### ISSUE TITLE
PayPal not adjusting to Free Shipping

### ISSUE THREAD

**Issue by Stickyweb** [woocommerce/woocommerce#18235]
I have WooCommerce set to free shipping and the customers receive an order confirmation showing Free Shipping, however, PayPal is still pulling in my original shipping rate of $18. PayPal say it is an issue with WooCommerce not sending the correct information.

**Comment by claudiosanches** [comment]
Hi there, thanks for posting. I wasn’t able to replicate this issue with the details you submitted. I tried placing an order with free shipping and it appeared to work correctly.

Are you sure that you are using PayPal integration that is included into WooCommerce? Maybe using a 3rd party plugin?

Please post back if you can provide more details on how to reproduce this issue. For now, I’m am closing this as non-reproducable.

Thanks

**Comment by Stickyweb** [comment]
@claudiosanches  Thank you for looking into this for me. After reviewing my PayPal setup, I realized during my PayPal set up, I also set up PayPal shipping methods. Thus, WooCommerce was correctly generating the correct invoice, adjusting for any WooCommerce shipping option, but when completing the PayPal payment, PayPal was forcing you to select the PayPal shipping method. Therefor after PayPal processed the payment, it was kicked back for a discrepancy between the totals. 

I've deleted the PayPal shipping methods, and everything is working normally.  Solved 👍
